### PR TITLE
add histogram example

### LINKED ISSUE #102: ssd1306 Histogram example
Hello,

I made this simple system monitor screen for my OrangePi Zero and ssd1306(128x64)
It shows a CPU load histogram , RAM usage bar, CPU  thermometer bar and system uptime.

I saw a histogram in "TODO" somewhere in your examples while learning how to use the library so i thought maybe you guys can use this in your repo.

I'm pretty much a newbie when it comes to programming so i'm sure this can be optimized

[sysinfo_ssd1306_128x64.zip](https://github.com/rm-hull/luma.examples/files/2970030/sysinfo_ssd1306_128x64.zip)

### PR DESCRIPTION
**Pull request by Tkopic001**
https://github.com/rm-hull/luma.examples/issues/102#issuecomment-473391918

I added a new file with my example system monitor

fixes #102

### CODE REVIEW

**Comment by thijstriemstra** [comment]
thanks @Tkopic001 

The travis build process detected some cosmetic issues, like whitespace around operators etc. Can you take a look and remove that whitespace?

See https://travis-ci.org/rm-hull/luma.examples/jobs/506936035#L361 for details.

**Comment by Tkopic001** [comment]
I'm not really sure what's going on, i've went trought the code 4 times removing all the spaces,

It runs fine on multiple machines:
1. My PC
2. OrangePI 
3. Cubieboard2

The details of the error look kinda strange:
https://travis-ci.org/rm-hull/luma.examples/jobs/506953487

**Comment by Tkopic001** [comment]
Oh, ok i see what's going on, i'm gonna fix it...

**Comment by Tkopic001** [comment]
Well this was annoying,
 but i finally fixed it,

**Comment by rm-hull** [comment]
Happy to take it into the examples repo but I will make work the same as the other examples (i.e. using get_device, so that it works across different display types)

File: examples/HistogramExample.py [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+def main():
+    # Importing some global vars
+    global blnk
+    global histogramData
+    global histogramTime
+    # Starting the canvas for the screen
+    with canvas(device, dither=True) as draw:
+        # Vars:
+        # Getting system uptime
+        sysUptime = timedelta(seconds=float(open('/proc/uptime', 'r').readline().split()[0]))
```
Comment (thijstriemstra) [inline examples/HistogramExample.py]: `/proc/uptime` should be closed after reading the file. use `with open(` etc instead.

**Comment by Tkopic001** [comment]
Sure, no problem, i'm happy you like it
I would love to adapt this to be a bit more dynamic and work across multiple displays and resolutions but only have this one monochrome SSD1306 display :/

**Comment by thijstriemstra** [comment]
The other examples show how to use get_device and make it compatible with all devices.

File: examples/HistogramExample.py (hunk 1 above)
Comment (Tkopic001) [reply examples/HistogramExample.py]: Oh, sorry for that.
I've only been typing python for a few days, will take a note on this

Comment (thijstriemstra) [reply examples/HistogramExample.py]: no worries!

**Comment by thijstriemstra** [comment]
Also add yourself to https://github.com/Tkopic001/luma.examples/blob/patch-1/CONTRIBUTING.rst

**Comment by rm-hull** [comment]
I made some small changes - it now works across a range of devices (inc. my mac), and we can use the **luma.emulator** devices now too:

luma_anim

File: examples/HistogramExample.py (hunk 1 above)
Comment (rm-hull) [reply examples/HistogramExample.py]: Fixed - used `psutil.boot_time()` instead

**Comment by rm-hull** [comment]
> can you also add it to the listing of examples in the docs?Will do that directly on master

**Pull request merged**